package.json

```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/UKHomeOffice/asl-pages#readme",
   "dependencies": {
-    "@asl/projects": "^15.5.15",
+    "@asl/projects": "^15.5.16",
```

Comment:
This will need to be 15.5.17 when asl-projects has been merged and tagged.

Also make sure to run `npm install` and commit the updated package-lock.json when updating dependencies, or `npm ci` won't work in the pipeline.

Reply:
updated to latest